portality/formcontext/formcontext.py

```diff
@@ -760,7 +760,11 @@ def finalise(self):
         # If this is the first time this application has been assigned to an editor, notify the publisher.
         old_ed = self.source.editor
         if (old_ed is None or old_ed == '') and self.target.editor is not None:
-            alerts = emails.send_publisher_editor_assigned_email(self.target)
```

Comment:
why don't we just decide which template and subject to send in the function `send_publisher_editor_assigned_email` rather than duplicating the send functions which are almost identical?

Reply:
They are almost identical but it has a different subject, different template and different receiver so basically all email parameters. Sure we can merge that method and just provide parameter but that's the case for pretty almost every email send (eg. inprogress, received, revisions_required or reject emails are also almost identical) so I believe it's a matter of convention and I followed the existing one.

Reply:
Excellent point :)